server/routes/index.js

```diff
@@ -0,0 +1,23 @@
+const router = require('express').Router();
+const config = require('../config');
+const path = require('path');
+const fs = require('fs');
+const httpStatus = require('http-status-codes');
+
+router.use(config.resources.videos, require('./videos'));
```

Comment:
The main router of the code is really the `app` (`app = express()`).
So all the `router.use` methods should be `app.use` in app.js file

Reply:
I think this is quite clean this way, every resource defines which sub resources are under him.
/ - 
router.use("/videos")
router.use("/health_check")
/videos
router.use("/:video_id")
...